### ISSUE TITLE
where can I find the category outputs for bottom-up features?

### ISSUE THREAD

**Issue by homelifes** [jackroos/VL-BERT#38]
Hello. Thanks for sharing your code
I downloaded `trainval2014_resnet101_faster_rcnn_genome` file from the google drive link. I could only find `image_id, image_w, image_h, num_boxes, boxes, features` inside it. But I can't find the predicted category for each box, in which you use in your masked ROI classification. May you tell me where are these located?

**Comment by jackroos** [comment]
The trainval2014_resnet101_faster_rcnn_genome is for VQA fine-tuning, not for pre-training, so we don't need the masked RoI classification.

**Comment by homelifes** [comment]
Sorry, missed that! Thanks a lot for your reply
